### PR TITLE
Migrate few more icon to sistent

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR migrate icons to sistent areas for now filters , patterns , metrics page
This migration i am doing in iterative process because there are many icons  and variant of  of mui icons to migrate and also i am making note which are not available that needed to be added in sistent parallelly

In this area icons needed to add in sistent:
```
doneicon
doneallicon
editicon
publish or public icon
infooutloined 
```
Above icons added here: https://github.com/layer5io/sistent/pull/563

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/ConfirmationModal.js [hunk 1]
```diff
@@ -134,6 +132,7 @@ const styles = (theme) => ({
         theme.palette.type == 'light'
           ? theme.palette.secondary.modalTabs
           : theme.palette.secondary.mainBackground,
+      fill: 'red',
```
Comment (sudhanshutech) [inline ui/components/ConfirmationModal.js:135]: oh, this need to be removed

File: ui/components/ConfirmationModal.js [hunk 2]
```diff
@@ -339,7 +341,7 @@ function ConfirmationMsg(props) {
                 className={classes.closeIconButton}
               >
                 <CloseIcon
-                  fill={'#FFFFF'}
+                  fill="#FFFFFF"
```
Comment (sudhanshutech) [inline ui/components/ConfirmationModal.js:344]: theme i will add

**Comment by leecalcote** [comment]
It looks like there's more to do before this is ready for review. I'm labeling with pr/draft. Remove this label and request review when ready.

**Comment by leecalcote** [comment]
Checking in..

**Comment by leecalcote** [comment]
Checking again...

**Comment by leecalcote** [comment]
Final check in before marking PR abandoned...

**Comment by sudhanshutech** [comment]
Oh,.my bad. I will soon update the pr and add.icons mentioned above

**Comment by leecalcote** [comment]
@sudhanshutech ready?

**Comment by sudhanshutech** [comment]
> @sudhanshutech ready?

yes

**Comment by leecalcote** [comment]
@sudhanshutech are these still not ready? Can they be moved forward a la carte?

**Comment by leecalcote** [comment]
merge conflicts...

**Comment by sudhanshutech** [comment]
> @sudhanshutech are these still not ready? Can they be moved forward a la carte?

yes it was ready , but i didn't merge because that time i was not merging. but i will remove conflicts and make it forward

**Comment by sudhanshutech** [comment]
icons updated here are already move as form sistent component. So this not required and further update in icon will do in subsequent pr

**Pull request closed without merging**